### ISSUE TITLE
Button missing text alternative on edit file page

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6811]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.1), the plus button next to `Select a Group` on the `Permissions` tab of the `Edit File` page does not have a text alternative and does not meet WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. As an admin, go to https://dev.nurax.samvera.org/concern/file_sets/p2676v71k/edit?locale=en#permissions_display and run the Site Improve browser extension
![file_button_missing_text_alt](https://github.com/samvera/hyrax/assets/32644254/3c41380b-775e-445a-b869-1257dd77f3d4)
